tests/test_utils.py

```diff
@@ -20,3 +43,70 @@ def test_get_s2_evalscript(all_bands: bool, reflectance: bool) -> None:
     assert input_units_str in evalscript
     output_sample_type_str = 'sampleType: "FLOAT32"' if reflectance else 'sampleType: "UINT16"'
     assert output_sample_type_str in evalscript
+
+
+pytestmark = pytest.mark.sh_integration
```

Comment:
dont mark the whole file, `test_get_s2_evalscript` does not need SH

Reply:
done